src/ui/server/helpers/display-number-field-value/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * helper to return number field value for mapping
+ * if number is 0, then will make 0 a string so can be displayed
+ * @param {Number} value
+ * @returns value - either number or string or null
+ */
+const displayNumberFieldValue = (value?: number) => {
+  if (value === 0) {
+    return String(value);
+  }
+
+  return value;
```

Comment:
Should we call this `transformNumberToString`, and always return a string? Can also then add a return type of `:string` to the function on line 7

Reply:
the return otherwise is null or number btw

Reply:
fails on eslint